File: Core/Entities/BudgetCeilingForCampaign.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    public class BudgetCeilingForCampaign : BaseEntity
    {
        public  double CampaignBudget { get; set; }
        public  double TotalExpense { get; set; }
        public  double TotalRemaining { get; set; }
        public int DonationId { get; set; }
        public string SBU { get; set; }
    }
}

File: client/src/app/shared/models/budgetCeilingForCampaign.ts

export interface IBudgetCeilingForCampaign {

    campaignBudget:  string;
    totalExpense:  string;
    totalRemaining: string;
    donationId: number;
    sbu: string;
}

export class BudgetCeilingForCampaign implements IBudgetCeilingForCampaign {
    campaignBudget:  string;
    totalExpense:  string;
    totalRemaining:  string;
    donationId: number;
    sbu: string;
}